src/renderers/controls/slider.control.tsx

```diff
@@ -0,0 +1,66 @@
+import { JSX } from '../JSX';
+import * as _ from 'lodash';
+import {
+  and,
+  optionIs,
+  or,
+  RankedTester,
+  rankWith,
+  schemaTypeIs,
+  uiTypeIs
+} from '../../core/testers';
+import { Control, ControlProps, ControlState } from './Control';
+import {
+    formatErrorMessage,
+    mapStateToControlProps,
+    registerStartupRenderer
+} from '../renderer.util';
+import { connect, Event } from '../../common/binding';
+import { resolveSchema } from '../../path.util';
+import { ControlElement } from '../../models/uischema';
+
+export const sliderControlTester: RankedTester = rankWith(4, and(
+    uiTypeIs('Control'),
+    or(schemaTypeIs('number'), schemaTypeIs('integer')),
```

Comment:
do we really need an uischema option `slider`?
Can't/Shouldn't we simply check for a `minimum` and `maximum` validation on the schema?
@edgarmueller what do you think?

Reply:
I am not sure how we can differentiate integer/number input field from a range slider if we dont say “I want this to be a slider”

Reply:
Yes, I agree, checking for `minimum` and `maximum` suffices.

Reply:
@AhmetTanakol The presence of `minimum` and `maximum` is enough to trigger the renderer.

Reply:
updated slider control tester and added new test cases for maximum and minimum validation